File: packages/ts/tests/e2e/__fixtures__/interpret/dirsql.config-nested.mjs
// Fixture: a native config that itself sets `config`. `dirsql interpret`
// must reject this -- a config file loaded by interpret cannot delegate to
// another config file. The referenced TOML is valid so the rejection comes
// from the loader, not a TOML read error.

import { dirname, join } from "node:path";
import { fileURLToPath } from "node:url";
import { DirSQL } from "dirsql";

const here = dirname(fileURLToPath(import.meta.url));

export default new DirSQL({ config: join(here, "nested.dirsql.toml") });

File: packages/python/tests/e2e/__fixtures__/interpret/dirsql.config_nested.py
"""Fixture: a native Python config that itself sets ``config=``.

`dirsql interpret` must reject this -- a config file loaded by interpret
cannot delegate to another config file (nested config loading). The
referenced TOML is valid, so the rejection comes from the loader, not from
a TOML read error.
"""

import os

from dirsql import DirSQL

app = DirSQL(config=os.path.join(os.path.dirname(__file__), "nested.dirsql.toml"))
